config/node/config.toml

```diff
@@ -34,7 +34,7 @@ IntervalToRefreshBlockedAddresses = "5m"
 IntervalToRefreshGasPrices = "5s"
 MaxTxBytesSize=100132
 MaxTxDataBytesSize=100000
-DefaultMinGasPriceAllowed = 0
+DefaultMinGasPriceAllowed = 1000000000
```

Comment:
if the initial gas is not 0, how does the first bridge tx work?

Reply:
maybe it was changed by the `make gasless off` and was committed.

Reply:
kk makes sense. yeah you can run gasless on again before commit